tests/generic_config_updater/test_pfcwd_status.py

```diff
@@ -190,9 +195,10 @@ def test_stop_pfcwd(duthost, extract_pfcwd_config, ensure_dut_readiness, port):
         4. Validates that orchagent is running fine pre and post test
     """
     pfcwd_config = extract_pfcwd_config
+    initial_count = len(pfcwd_config) * FLEXDB_COUNTERS_PER_PORT
 
     if port == 'single':
-        expected_count = get_flex_db_count(duthost) - 3
+        expected_count = initial_count - FLEXDB_COUNTERS_PER_PORT
```

Comment:
Should we wait_until `get_flex_db_count` finish instead of using the initial count? I saw you mentioned 

> To fix a case when we read the initial number of counters in flex_db at the moment they are added partially.

Should we wait for flex_db update finish?

Reply:
`get_flex_db_count` retrieves the current number of pfcwd entries, but doesn't ensure it matches with expected count.
In this test we use the `ensure_dut_readiness` fixture, that runs `check_config_update` to await until the actual number of pfcwd entries is the same as we expect(3 per each port).
I use `initial_count` as expected number of entries here, because it already the same as actual number.